salesforce/latest/using/community.md

```diff
@@ -0,0 +1,135 @@
+
+## Share content externally to community members Lightning Experience
+
+1. Log in to Salesforce.
+
+2. Log in as a community administrator to Alfresco Share in the Canvas app.
+
+      ![lightning-alfresco-tab]({% link salesforce/images/lightning-alfresco-tab.png %})
+
+3. Click the down arrow next to **Cases** and then **New Case**.
+
+      ![lightning-create-case]({% link salesforce/images/lightning-create-case.png %})
+
+4. Select the required **Case Origin** and any other information needed for the case. Click **Save**.
+
+      ![lightning-new-case]({% link salesforce/images/lightning-new-case.png %})
+
+5. Click the drop down arrow next to **Change Owner** and select **Sharing**.
+
+      ![create-new-lightning-page]({% link salesforce/images/lightning-sharing.png %})
+
+6. Click the drop down arrow in the search bar and select **Public Group**.
+
+      ![lightning-public-group]({% link salesforce/images/lightning-public-group.png %})
+
+7. Type *all* into the search field and select **All Customer Portal Users**.
+
+      ![lightning-portal-users]({% link salesforce/images/lightning-portal-users.png %})
+
+8. Click the **Case Access** field and select the type of access required for this case. Click **Save**.
+
+      ![lightning-share-access]({% link salesforce/images/lightning-share-access.png %})
+
+9. Click the **Create** drop down list and select **Folder**.
+
+      ![lightning-create-folder]({% link salesforce/images/lightning-create-folder.png %})
+
+10. Enter the details you require for your new folder and then click **Create**.
+
+      ![lightning-details-folder]({% link salesforce/images/lightning-details-folder.png %})
+
+11. Click the **Actions** drop down list and select **Share Externally**.
+
+    ![lightning-share-externally]({% link salesforce/images/lightning-share-externally.png %})
+
+12. Select the **Salesforce Community Name** you would like to share with and the **Users**. Click **Save**.
+
+      ![lightning-externally-select]({% link salesforce/images/lightning-externally-select.png %})
+
+You will see a **Successfully Shared Accounts Externally** message. The result of the performed actions is that when the community member logs into Salesforce they will see the case in their **All Open Cases** list.
+
+## Salesforce Classic
+
+1. See [Create an Experience Cloud Site pg. 51](https://resources.docs.salesforce.com/latest/latest/en-us/sfdc/pdf/communities.pdf){:target="_blank"}
+
+2. See [Add Members to Your Experience Cloud Site pg. 54](https://resources.docs.salesforce.com/latest/latest/en-us/sfdc/pdf/communities.pdf){:target="_blank"}
+
+3. [Share content externally to community members Salesforce Classic](#share-content-externally-to-community-members-salesforce-classic)
+
+## Share content externally to community members Salesforce Classic
+
+1. Log in to Salesforce.
+
+2. Click **Alfresco Repository** and log in as a community administrator to Alfresco Share in the Canvas app.
```

Comment:
Remove word "community".
.... log in as an administrator to Alfresco Share in the Canvas app.

Reply:
done